cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -418,6 +422,28 @@ def setUp(self):
         # Create a second sequential just (testing children of children)
         self.create_xblock(parent_usage_key=self.chapter_usage_key, category='sequential2')
 
+    def test_discussion_duplication(self):
+        """
+        Tests that duplicated discussion component have unique 'discussion_id',
+        if 'discussion_id' is present in xml.
+        """
+        store = modulestore()
+        # Importing course from xml
+        courses = import_course_from_xml(
```

Comment:
Is it possible to add the `discussion_id` to the discussion item created with `ItemFactory` before you duplicate it?